notification/src/handler/notification/notification.handler.js

```diff
@@ -112,6 +132,21 @@ async function updatePaymentWithRepeater(
   }
 }
 
+function _obfuscateNotificationInfoFromActionFields(updateActions) {
```

Comment:
Why do we need that? What do we attempt to filter out?

Reply:
`addInterfaceInteraction` contains a whole notification object which could sensitive data in it, before we decided to not log all details of the notification payload. Here I figured out this part is not fixed, so added it as a boy scout kind of change.

Reply:
So what useful data it could potentially log if we remove all notification details?

Reply:
One example might be, check the message, also in controller we put the notification into error context as it might be a different issue.

````
{
  "name": "ctp-adyen-integration-notifications",
  "hostname": "ct-00891.local",
  "pid": 90276,
  "level": 50,
  "notification": [
    {
      "eventCode": "AUTHORISATION",
      "eventDate": "2019-01-30T18:16:22+01:00",
      "pspReference": "test_AUTHORISATION_1",
      "success": "true"
    }
  ],
  "err": {
    "message": "Got a concurrent modification error when updating payment with id \"undefined\". Version tried \"undefined\", currentVersion: \"2\". Won't retry again because of a reached limit 20 max retries. Failed actions: [{\"action\":\"addInterfaceInteraction\",\"type\":{\"key\":\"ctp-adyen-integration-interaction-notification\",\"typeId\":\"type\"},\"fields\":{\"createdAt\":\"2021-03-25T06:40:33.161Z\",\"status\":\"authorisation\",\"type\":\"notification\",\"notification\":{\"eventCode\":\"AUTHORISATION\",\"eventDate\":\"2019-01-30T18:16:22+01:00\",\"pspReference\":\"test_AUTHORISATION_1\",\"success\":\"true\"}}},{\"action\":\"changeTransactionState\",\"transactionId\":\"9ca92d05-ba63-47dc-8f83-95b08d539646\",\"state\":\"Success\"}]",
    "isRecoverable": true
  },
  "msg": "Unexpected exception occurred.",
  "time": "2021-03-25T06:41:07.653Z",
  "v": 0
}


````

Reply:
I see thanks. I wonder is it just the test or the payment id might be always "undefined"?

Reply:
No it's by chance, mock does not have both version and id, mock is a draft :)